src/rgw/driver/rados/rgw_bucket_sync.cc

```diff
@@ -993,6 +995,19 @@ void RGWBucketSyncPolicyHandler::get_pipes(std::set<rgw_sync_bucket_pipe> *_sour
   }
 }
 
+bool RGWBucketSyncPolicyHandler::bucket_exports_object(std::string obj_name, RGWObjTags* tags) {
```

Comment:
please pass the string by const ref. if the `tags` is not an optional argument, pass that by const ref too